app/views/layouts/_top_menu.html.erb

```diff
@@ -15,8 +15,9 @@
           <li class="dropdown">
             <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button">People<span class="caret"></span></a>
             <ul class="dropdown-menu">
-              <%= nav_link "Police", police_people_path %>
-              <%= nav_link "CERT", cert_people_path %>
+              <% Department.managing_people.order("shortname ASC").each do |dept| %>
+                <%= nav_link dept.shortname, department_people_path(dept) %>
```

Comment:
Any advice on how to do this? I can add a migration to enforce it at the db level, but it would rollback if any existing rows would fail the new constraint. Should I copy a default value over from the name as part of the migration, if necessary? Or would the policy be to operate as if all the data was already valid? I think even the seeds.rb file for example neglects adding shortname.